### ISSUE TITLE
Add accessible `tabpanel` role to the panels handled by selection tabs in the sidebar

### ISSUE THREAD

**Issue by acelaya** [hypothesis/client#6267]
The sidebar shows up to three tabs for "Annotations", "Page notes" and "Orphans". All of them have the right `role="tab"`. However, the panels they handle are missing the `role="tabpanel"` attribute.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role
